openreview/journal/journal_request/journal_request.py

```diff
@@ -168,7 +168,7 @@ def setup_comment_invitation(self, note_id, action_editors_id=None):
                 writers = [],
                 signatures = ['~Super_User1'],
                 edit = {
-                    'signatures': { 'regex': f'~.*|{self.support_group_id}', 'type': 'group[]' },
+                    'signatures': { 'regex': f'~.*|{venue_id}|{self.support_group_id}', 'type': 'group[]' },
```

Comment:
had to add this here to allow the `venue_id` to sign the comment (since we couldn't sign it using `support_group.id` from the process function)

Reply:
I think it is ok that the venue_id can sign that note.